src/servers/src/query_handler/sql.rs

```diff
@@ -37,6 +38,13 @@ pub trait SqlQueryHandler {
         query_ctx: QueryContextRef,
     ) -> Vec<std::result::Result<Output, Self::Error>>;
 
+    async fn execute_plan(
```

Comment:
For the consistency of trait methods names, I suggest naming this method to `do_logical_plan`.

Besides, does this method has to take `query: &str`? If the query don't match the plan, what's the implementor's behavior and how can we ensure that? There's ambiguity in it.

Reply:
I prefer `do_exec_plan`.  `do_logical_plan` is weird.